apps/explorer/src/ui/DescriptionList.tsx

```diff
@@ -1,17 +1,34 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import clsx from 'clsx';
+
 import type { ReactNode } from 'react';
 
 export interface DescriptionItemProps {
     title: string | ReactNode;
     children: ReactNode;
+    border?: boolean;
 }
 
-export function DescriptionItem({ title, children }: DescriptionItemProps) {
+export function DescriptionItem({
+    title,
+    border,
+    children,
+}: DescriptionItemProps) {
     return (
-        <div className="flex flex-col gap-2 md:flex-row md:items-center md:gap-10">
-            <dt className="w-full flex-shrink-0 text-pBody font-medium text-steel-darker md:w-50">
+        <div
+            className={clsx(
+                'flex flex-col gap-2 md:flex-row md:items-center md:gap-10',
+                border && 'rounded-xl border border-success px-4 py-2'
```

Comment:
This feels _way_ too specific to the one use-case to be in here. Should be in the parent or as a one-off.

Reply:
removed it to use one of